packages/medusa/src/api/routes/admin/price-lists/update-price-list.ts

```diff
@@ -267,4 +268,12 @@ export class AdminPostPriceListsPriceListPriceListReq {
     IsBoolean(),
   ])
   includes_tax?: boolean
+
+  constructor() {
+    if (
+      !featureFlagRouter.isFeatureEnabled(TaxInclusivePricingFeatureFlag.key)
+    ) {
+      delete this.includes_tax
+    }
```

Comment:
as seen together, we can now remove that everywhere

Reply:
done ✅